index.html

```diff
@@ -801,6 +872,56 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.
         <div class="container">
 
 
+<!-- legacy releases v4.0.20 -->
+<!-- 
+    <div class="break"></div> 
+    insert this code when a new update comes out to separate a new version from the current.
+-->
+<div class="smaller-margin" id="lr-12">
+    <h2 class="text-middle">Legacy releases</h2>
+    <!-- h2 tag needs to be moved to the newest version -->
+    <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.0.19...v4.0.20">v4.0.20</a></h4>
+</div>
+
+<div class="dimmed-text"><p>Release date: 2018-09-28</p></div>
+
+<ul>
+    <li>Fixed/added:</li>
+    <ol>
+        <li>Updated sortbootorder to v4.6.11</li>
+    </ol>
+    <li>Known issues:</li>
+    <ol>
+        <li>pfSense 2.4.x fails to boot when no USB stick is plugged</li>
```

Comment:
@Moviss I tested v4.0.20 and had no reproduction of the pfSense issue. This should be removed.